src/jdk.jartool/share/classes/sun/tools/jar/Main.java

```diff
@@ -875,6 +875,7 @@ void create(OutputStream out, Manifest manifest) throws IOException
                 }
                 ZipEntry e = new ZipEntry(MANIFEST_DIR);
                 setZipEntryTime(e);
+                e.setMethod(ZipEntry.STORED);
```

Comment:
This should be okay, consistent with addFile where it will use STORED for directories.